packages/core/src/domain/configuration/configuration.ts

```diff
@@ -131,6 +133,12 @@ export function validateAndBuildConfiguration(initConfiguration: InitConfigurati
     return
   }
 
+  if (initConfiguration.usePciIntake === true && initConfiguration.site !== 'datadoghq.com') {
```

Comment:
**🔨 warning:** ‏at this point `initConfiguration.site` is whatever the user provided, so `site` has no default yet. The condition should be:
```suggestion
  if (initConfiguration.usePciIntake === true && initConfiguration.site && initConfiguration.site !== 'datadoghq.com') {
```

Reply:
Done ✔️